src/storage/temporary_file_manager.cpp

```diff
@@ -253,14 +252,13 @@ bool TemporaryFileIndex::IsValid() const {
 //===--------------------------------------------------------------------===//
 
 static idx_t GetDefaultMax(const string &path) {
-	// Use the available disk space
+	D_ASSERT(!path.empty());
 	auto disk_space = FileSystem::GetAvailableDiskSpace(path);
-	idx_t default_value = 0;
-	if (disk_space.IsValid()) {
-		// Only use 90% of the available disk space
-		default_value = static_cast<idx_t>(static_cast<double>(disk_space.GetIndex()) * 0.9);
-	}
-	return default_value;
+	// Use the available disk space
+	// We have made sure that the file exists before we call this, it shouldn't fail
+	D_ASSERT(disk_space.IsValid());
```

Comment:
Can we check this again? There is likely going to be some random file system for which `statvfs` fails on some random linux distribution